pdns/recursor_cache.cc

```diff
@@ -633,7 +649,7 @@ uint64_t MemRecursorCache::doDump(int fd)
       for (const auto &sig : i.d_signatures) {
         count++;
         try {
-          fprintf(fp.get(), "%s %" PRId64 " IN RRSIG %s ; %s\n", i.d_qname.toString().c_str(), static_cast<int64_t>(i.d_ttd - now), sig->getZoneRepresentation().c_str(), i.d_netmask.empty() ? "" : i.d_netmask.toString().c_str());
+          fprintf(fp.get(), "%s %" PRIu32 " %" PRId64 " IN RRSIG %s ; %s\n", i.d_qname.toString().c_str(), i.d_orig_ttl, static_cast<int64_t>(i.d_ttd - now), sig->getZoneRepresentation().c_str(), i.d_netmask.empty() ? "" : i.d_netmask.toString().c_str());
```

Comment:
I'm not sure we care about the original TTL for RRSIGs since we already dump it for the entry itself.

Reply:
But doesn't the same hold for the actual ttl that was already there?

Reply:
It does indeed, and for the netmask as well. Perhaps it makes sense to stop dumping these? Or we can keep the code as it is, it's not a big issue. I guess the only drawback is making the dump bigger, and perhaps a bit longer to generate.

Reply:
I decided to keep the current code, it makes it a tad more easy to spot which lines belong to the same cache entry. We could make the dump more compact by having a entry header en then the individual records without repeating info and then a clear end of entry marker, but that's for some other time.